### ISSUE TITLE
qt5-base fails to build on linux.

### ISSUE THREAD

**Issue by Knitschi** [microsoft/vcpkg#5038]
I tried to install the qt5-base package on a linux debian system and the build failed.

```
Package: qt5-base:x64-linux
Vcpkg version: 2018.11.23-unknownhash
```

The build fails with the following err and out logs.
[build-x64-linux-dbg-err.log](https://github.com/Microsoft/vcpkg/files/2710183/build-x64-linux-dbg-err.log)
[build-x64-linux-dbg-out.log](https://github.com/Microsoft/vcpkg/files/2710184/build-x64-linux-dbg-out.log)

Cheers to the busy bee who carries the burden of maintaining the qt package ;)

**Comment by TheScarfix** [comment]
can you try it with my qt5.12 branch? https://github.com/TheScarfix/vcpkg/tree/qt-5.12

**Comment by JackBoosY** [comment]
Hi @Knitschi, thanks for reporting this issue.
This issue seems have been fixed by PR #5293, and I can't repro it.

Please update vcpkg and rebuild qt5-base.

Thanks.
